extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.js

```diff
@@ -491,6 +491,20 @@
 		_t.buttonDown = _t.parentNode.querySelector(o.rollerblind.down);
 		_t.buttonStop = _t.parentNode.querySelector(o.rollerblind.stop);
 
+		_t.hasValue = _t.parentNode.getAttribute("data-has-value") === "true";
+		_t.valueNode = _t.parentNode.parentNode.querySelector(o.formValue);
+		_t.setValuePrivate = function(value) {
+			if (!_t.hasValue) {
+				return;
+			}
+			if (value === "DOWN") {
```

Comment:
Well, the RollershutterItem defines UP/DOWN as accepted values for its state: https://github.com/eclipse/smarthome/blob/master/bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/items/RollershutterItem.java#L42
This has been like this since the very beginning.
We can discuss whether this should be considered to be a bug and we should only allow UNDEF or a PercentType instead - it would make things indeed easier for consumers such as the Basic UI. The problem is that such a change would break the auto update feature, because this is the one which currently sends the UP update when an UP command is sent for the item.